### ISSUE TITLE
Output truncated?

### ISSUE THREAD

**Issue by pawamoy** [paulirish/github-email#16]
Hi, are the output or emails truncated? One of mine misses the last `m` of `.com`.

The username is 22 chars long, then there is the padding, then the email is 36 chars long.

```
Emails from owned-repo recent activity
*********************		    *********************@**********.co
```

I checked my local git clones for emails (`for repo in $(all-repo); do (cd $repo && git log --format='%ae%n%aE%n%ce%n%cE'); done | sort -u`) and couldn't find the `.co` email (the `.com` one is correctly listed).

**Comment by paulirish** [comment]
heh thanks for noticing. 

https://github.com/paulirish/github-email/blob/master/github-email.sh#L68

^ defaults to a width of 72. I'll up it a bit.

**Comment by paulirish** [comment]
it's a lame fix but i'm short on time to do something better.

shipped in 1.1.1

**Comment by pawamoy** [comment]
Good, thanks :slightly_smiling_face:
